### PR TITLE
Add recipe for blend_modes

### PR DESCRIPTION
**Pull request by flrs**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/blend_modes/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,81 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
```
Comment (scopatz) [inline recipes/blend_modes/meta.yaml]: Please remove comments

Comment (flrs) [reply recipes/blend_modes/meta.yaml]: Done! Thanks for reviewing!

File: recipes/blend_modes/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 0a3145e4792e005764b9663f5ce899d30f7c24f4bcff00428907d03dbe068f37
+
+build:
+  number: 0
+  skip: true  # [py<35]
+  script: "{{ PYTHON }} -m pip install . -vv"
```
Comment (scopatz) [inline recipes/blend_modes/meta.yaml]: can this be noarch python?

Comment (flrs) [reply recipes/blend_modes/meta.yaml]: Yes. The only Python version requirement is >= 3.5.

I followed the [maintainer knowledge base on conda-forge.org](https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-builds) which mentions explicitly as noarch requirement:
> No skips except for python version. If the recipe is py3 only, remove skip statement and add version constraint on python in host and run section.

I removed the skip statement and added version constraints here:

https://github.com/conda-forge/staged-recipes/blob/e9c968f236e38e620252a0ccec7cc19f88a6981c/recipes/blend_modes/meta.yaml#L17-L25

**Pull request merged**